docs/src/pages/components/drawers/ResponsiveDrawer.js

```diff
@@ -51,7 +51,7 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export default function ResponsiveDrawer(props) {
+function ResponsiveDrawer(props) {
```

Comment:
I'm not sure.

cc @oliviertassinari

Reply:
I couldn't find much material on mutable exports, the only one I could find was referring to referrence changes as "mutable exports". In this case, the object exported has the same reference. Maybe we should call that a "weak mutable export"?

Does it have a negative effect in practice? The only pros I'm aware of in the previous approach is that it requires fewer LOCs.

Reply:
As I said more academic. IMO once you hit `export` in an actual module the exported member shouldn't change anymore because you get all sorts of indirection otherwise. But that's a rather strict interpretation. Just want to caution that we won't go overboard with it.

Reply:
@merceyz How much would it cost (in time) to change the JavaScript output format? It's not incredibly important. I think anything above 1 hour could self qualified as a low return.

Reply:
Depends, which part are you thinking of?

Reply:
I was thinking of changing the logic to have no diff on this line.

Reply:
Not much, just need to add a option to not perform the action on this line
https://github.com/merceyz/typescript-to-proptypes/blob/232634f7782a8119d012ceb69b27e30911ff8052/src/injector.ts#L257

Though I don't like the idea of changing something after it's exported

Reply:
> Though I don't like the idea of changing something after it's exported

I share the same feeling.

Reply:
So we keep it as it in the diff? If we want to use it for the core components, it would be better to play it safe, even if only has a psychological (it's a good reason enough).

Reply:
> So we keep it as it in the diff? 

Yeah